aptos-move/move-examples/drand/Move.toml

```diff
@@ -10,4 +10,5 @@ AptosFramework = { local = "../../framework/aptos-framework" }
 std = "0x1"
 aptos_std = "0x1"
 aptos_framework = "0x1"
+source_addr = "_"
```

Comment:
Would it be better to rename this to `developer_address`. I find "developer" more informative than "source" and in also in line with [the lottery PR](https://github.com/aptos-labs/aptos-core/pull/9581/files#diff-6f8716d830670b98d3861da5eec4442bfd3e2f6db3b35e340bb63c146ac05729R14))

Reply:
Replaced with `developer_addr`